### DISCUSSION TITLE
How do I package external PDB files into msixsym

### DISCUSSION THREAD

**Discussion by levinli303** [microsoft/WindowsAppSDK#1952] [Q&A] [answered]
Hi, I have a WinUI app that references some Win32 DLLs that I built outside Visual Studio, it runs well. When I package the app from Visual Studio, it packages the symbols files (PDB) into a msixsym, and since these DLLs are external, their PDBs are not copied into the msixsym file. How do I make VS package these PDB files as well?

I tried unzipping the msixsym, adding the PDB files, re-zipping into a msixsym file, (and subsequently put together with a .msixbundle file into a .msixupload) but when I uploaded to Microsoft Partner Center, it show error:

Package acceptance validation error: The file MyAppName_MyAppVersion_x64.msixsym is not supported. Remove or update this file and then try again.

What is the correct way to package these external PDBs in a msixsym?

**Comment by riverar** [comment, marked as answer]
Rename them to `.appxsym`. `.msixsym` isn't supported within `.msixupload` last I checked.

**Reply by levinli303** [reply]
Thanks, that solves it! Although my memory tells me that I once uploaded an .msixupload successfully containing .msixsym that generated by visual studio.
